README.md

```diff
@@ -166,16 +193,16 @@
     ```
 
 
-    **[[⬆]](#TOC)**
+**[⬆ back to top](#table-of-contents)**
 
 
-## <a name='functions'>Functions</a>
+## Functions
 
   - Function expressions:
 
     ```javascript
     // anonymous function as event handler
-    $el.on('click', function () {
```

Comment:
The space is required. Having a space between `function` and `()` helps to indicate that "function" is not the _name_ of the function, and that this is an un-named anonymous function.